modules/ephemeral-storage-using-a-sharedconfigmap-object-in-a-pod.adoc

```diff
@@ -7,19 +7,19 @@
 [id="ephemeral-storage-using-a-sharedconfigmap-object-in-a-pod_{context}"]
 = Using a SharedConfigMap instance in a pod
 
-To access a `SharedConfigMap` custom resource (CR) instance from a pod, you grant a given service account role-based access control (RBAC) permissions to use that `SharedConfigMap` CR instance.
+To access a `SharedConfigMap` CR instance from a pod, you grant a given service account role-based access control (RBAC) permission to use that `SharedConfigMap` CR instance.
```

Comment:
Since this is the first instance in a file, you can expand CR to custom resource.

Reply:
To access a `SharedConfigMap` custom resource (CR) from a pod, you must grant the associated service account the necessary Role-based access control (RBAC) permissions to use that `SharedConfigMap` CR.